modules/multi-arch-gather-info-about-workloads.adoc

```diff
@@ -0,0 +1,31 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="multi-architecture-gather-info-about-workloads_{context}"]
+= Gathering information about the workloads in your multi-architecture cluster
+
+After installing the Multiarch Tuning Operator, you can verify the multi-architecture support for workloads in your cluster. When you create a pod, the Multiarch Tuning Operator adds pod labels that allow you to identify and manage pods based on their architecture compatibility. These labels are automatically set on the newly created pods to provide insights into their architecture support.
```

Comment:
Can we remove 'When you create a pod,'?

Reply:
Updated as per the suggestion. @aleskandro